style.md

```diff
@@ -1515,6 +1516,114 @@ Note that the compiler will not generate errors when using predeclared
 identifiers, but tools such as `go vet` should correctly point out these and
 other cases of shadowing.
 
+### Avoid `init()`
+
+Avoid `init()` where possible. At a minimum, all code within `init()` should:
+
+1. Be completely deterministic, regardless of program environment or invocation.
+2. Be completely self-contained within the program.
```

Comment:
What do you mean by self-contained -- no dependencies?

Reply:
Yeah, but I don't think this provides value over "no IO", et al. I'll remove it.